### PR TITLE
Add beautify extension

### PR DESCRIPTION
**Pull request by HunseopJeong**
It makes minified script to be beautified code.

Signed-off-by: Hunseop Jeong <hs85.jeong@samsung.com>

### CODE REVIEW

File: libs/brackets-server/embedded-ext/hirse.beautify/default.jsbeautifyrc [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        "brace_style": "collapse",
+        "keep_array_indentation": true,
+        "keep_function_indentation": false,
+        "space_before_conditional": true,
+        "break_chained_methods": false,
+        "eval_code": false,
+        "unescape_strings": false,
+        "wrap_line_length": 0,
+        "wrap_attributes": "auto",
+        "end_with_newline": true
```
Comment (Jw00) [inline libs/brackets-server/embedded-ext/hirse.beautify/default.jsbeautifyrc]: Do these items need to be sorted?

File: libs/brackets-server/embedded-ext/hirse.beautify/main.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+        case 'less':
+            beautifierType = 'css';
+            break;
+        default:
+            Dialogs.showModalDialog(DefaultDialogs.DIALOG_ID_ERROR, Strings.UNSUPPORTED_TITLE, Strings.UNSUPPORTED_MESSAGE);
+            return;
+        }
+
+        var unformattedText;
+        var editor = EditorManager.getCurrentFullEditor();
```
Comment (Jw00) [inline libs/brackets-server/embedded-ext/hirse.beautify/main.js]: These two variables seems to be better positioned line 92, using them.

File: libs/brackets-server/embedded-ext/hirse.beautify/default.jsbeautifyrc (hunk 1 above)
Comment (HunseopJeong) [reply libs/brackets-server/embedded-ext/hirse.beautify/default.jsbeautifyrc]: Fixed.

File: libs/brackets-server/embedded-ext/hirse.beautify/main.js (hunk 2 above)
Comment (HunseopJeong) [reply libs/brackets-server/embedded-ext/hirse.beautify/main.js]: Fixed.

**Comment by Jw00** [comment]
LGTM

**Pull request merged**